### PR TITLE
changes for codecov

### PR DESCRIPTION
**Pull request by TriptiTripathi1234**

### CODE REVIEW

File: .github/workflows/pr-test.yml [hunk 1]
```diff
@@ -1,20 +1,20 @@
 name: test
 on:
   push:
-  pull_request_target: 
+  pull_request_target:
     branches:
-      - main  
+      - main
```
Comment (jaywalker21) [inline .github/workflows/pr-test.yml]: Shouldn't this be `rzp_main`?

**Review by jaywalker21 — COMMENTED** [review]
@TriptiTripathi1234 
Can you change branch name in `pr-test.yml` and `pr-build.yml` to `rzp_main`?

File: .github/codecov.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+codecov:
+  branch: rzp_main # Default branch
```
Comment (13shivam) [inline .github/codecov.yml]: this is not required, if using codecov/codecov-action@v2 in pr-test.yml

Comment (jaywalker21) [reply .github/codecov.yml]: For me, codecov was taking default branch as `main` for some reason, even though we had `rzp_main`.
Had to put this to have it in sync.

Comment (TriptiTripathi1234) [reply .github/codecov.yml]: before this change, it was failing and report was not added in PR comment. Check out [here](https://app.codecov.io/gh/razorpay/hypertrace-ingester/commits?branch=adding_codecov)

Comment (13shivam) [reply .github/codecov.yml]: i think pr-test was running on main branch, did you try rzp_main in pr-test.yml?

File: .github/workflows/pr-test.yml (hunk 1 above)
Comment (TriptiTripathi1234) [reply .github/workflows/pr-test.yml]: done

File: .github/codecov.yml (hunk 2 above)
Comment (TriptiTripathi1234) [reply .github/codecov.yml]: yes still same issue

Comment (TriptiTripathi1234) [reply .github/codecov.yml]: after merge, the code will be present in rzp_main, then i think it will work as expected

**Review by jaywalker21 — APPROVED** [review]

**Pull request merged**